.internal/templates/services/nodered/template.yml

```diff
@@ -1,24 +1,16 @@
 nodered:
   container_name: nodered
-  build: ./services/nodered/.
+  build:
+    context: ./services/nodered/.
+    args:
+      - DOCKERHUB_TAG=latest
```

Comment:
Should we set a specific version? Newer versions might break something (To be fair, NodeRed has been fairly good though).

Reply:
Well, this mechanism whereby users can choose the version in the service definition (rather than editing the Dockerfile in the services directory) has been around for a while, at least for master and old-menu where the corresponding PRs are just fixing issues flagged by `yamllint`. In that sense, this one on experimental was *intended* to be a catch-up but I'm now suddenly wondering about the associated Dockerfile template.

🤬🤦

Yup, applying this PR would break the build because I didn't change the Dockerfile template to support `DOCKERHUB_TAG`.

On balance, rather than try to change the Dockerfile template (which I can't test), it is probably better for me to just close this PR. Sorry.